docs/assets/2.13.-2.14-cleanup-jaeger.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+kubectl delete -n kyma-system tracepipelines.telemetry.kyma-project.io jaeger
+kubectl delete -n kyma-system telemetries.telemetry.istio.io kyma-traces
```

Comment:
Why no review happened here from a developer?
The resource is not placed in the kyma-system namespace and the command will fail, see https://github.com/kyma-project/kyma/blob/2.13.0/resources/tracing/templates/kyma-additions/istio-telemetry.yaml#L6

Why cluster scoped resources are using a namespace in he script?
Also the migration guide related to the script is missing, see for example https://kyma-project.io/docs/kyma/2.12/migration-guide-2.11-2.12

Reply:
I, too, was wondering why the only mandatory review was by TW. 
Is the code owners assignment wrong?